src/Moat.hs

```diff
@@ -44,6 +44,9 @@ module Moat
     -- ** Option type and defaults
```

Comment:
The changes here are just passing options down to `enumExp`

Reply:
i.e. https://github.com/MercuryTechnologies/moat/pull/35/files#diff-eaefe7c4f367e718fa37d0be56ad7b47a636be3046346d30d9494ccd555625f7R1490